CVE-2017-12617/cvex.yml

```diff
@@ -0,0 +1,14 @@
+blueprint: ubuntu2204
+ubuntu:
+  trace: nginx
+  playbook: ubuntu.yml
+  command:
+    - "sudo chown -R vagrant:vagrant /home/vagrant/apache-tomcat-9.0.0.M1"  
```

Comment:
All these commands (except of the exploit execution) belong to the ansible playbook. The intention is to have a playbook that installs and configures the server. Please move them to the playbook.